### ISSUE TITLE
For each API endpoint in api.ts, create a GH Issue for switch the FE to use the new Hasura endpoint.

### ISSUE THREAD

**Issue by crabsinger** [coordinape/coordinape#409]
Once the work is complete to migrate functionality from Laravel to Hasura (#403), the next step is to switch the frontend to use the new Hasura endpoint.

For each api endpoint (see work from issue #403), create a Github Issue to use the new Hasura GraphQL api endpoint in the frontend app.

1. Use Zeus for all GraphQL interactions
2. Use ReactQuery for all new components, and try to refactor to use it removing Recoil if easy.
3. Fall back to not using ReactQuery and just using Zeus and Recoil if necessary to complete Hasura migration, and then refactor later.

The more challenging part will be the mutations which do not use recoil, they just trigger a global refresh.

**Comment by crabsinger** [comment]
Would be nice to have a consistent issue template for all tasks that are created from this effort.

**Comment by topocount** [comment]
>     1. Use Zeus for all GraphQL interactions
> 
>     2. Use ReactQuery for all new components, and try to refactor to use it removing Recoil if easy.
> 
>     3. Fall back to not using ReactQuery and just using Zeus and Recoil if necessary to complete Hasura migration, and then refactor later.

@CryptoGraffe I think the consensus from our last meeting was that component-level state requests using `react-query` should be out of scope to finish this migration, and we could introduce it with the V2 frontend. Essentially, we should just continue using recoil because the lift right now probably isn't worth the return.

**Comment by levity** [comment]
Fwiw, Tony and I are trying to use Recoil as little as possible as we build out the vaults UI. Data fetching with GraphQL is just a lot easier to understand

**Comment by CryptoGraffe** [comment]
This is done
